src/controllers/satisfactioncontroller.js

```diff
@@ -0,0 +1,47 @@
+import Controller from './core/controller';
+import CustomerController from './customercontroller';
+import ScoreController from "./scorecontroller";
+
+export default class SatisfactionController extends Controller
+{
+    /**
+     * Updates player satisfaction after every completed order, as well as the score
+     * @param customer
+     */
+    updatePlayerSatisfaction(customer)
+    {
+        if (customer.satisfaction > 100)
+            customer.satisfaction = 100;
+
+        let config = GAME.model.config;
+
+        config.playerSatisfaction = ((config.playerSatisfaction * config.completedOrders) + (customer.satisfaction))
+                                    / (config.completedOrders + 1);
+
+        $("#satisfaction").html(config.playerSatisfaction.toFixed(0));
+
+        const scoreController = new ScoreController();
+
+        scoreController.updateScore();
+    }
+
+    /**
+     * Updates satisfaction of customers each time a new day occurs, and adds appropriate emojis.
+     */
+    updateCustomerSatisfaction()
+    {
+        const customerController = new CustomerController();
+
+        for (let customer of GAME.model.customers) {
+            customer.satisfaction -= 10;
+
+            if (customer.satisfaction <= GAME.model.config.sendAwayThreshold)
+            {
+                this.updatePlayerSatisfaction(customer);
+                customerController.sendAway(customer);
+            }
+        }

```

Comment:
Where do you now determine which emoji has to be loaded? It seems as the default emoji is not changed anywhere (which might make it look like your view is not updating). Or am I overlooking something?

Reply:
No this should be done by the update customer view in the customer controller, then base the template to be loaded on the customer satisfaction. Not sure how to work with that though. If you have pointers/links to how to work with that I can hopefully finish it tomorrow. Shouldn't be too difficult, so maybe you could even do it. Of course, you may not have the time either.